.eslintrc

```diff
@@ -171,7 +171,9 @@
       "off"
     ],
     "no-unused-vars": [
-      "off"
+      "error", {
+        "vars": "all"
```

Comment:
I read here for the options of this: https://eslint.org/docs/rules/no-unused-vars#options. By default for `args` it is `"args": "after-used"`. Why are we not using `"args": "all"`?

Reply:
OK, so we've certain terms like `$scope.$on(EVENT_NEW_CARD_OPENED, function(evt, newCard)` where evt is never used but by default it would be the first argument.

So, let's proceed with the default behaviour. Could you please also add `"args": "after-used"` as an option in this rule?